File: MySocialNetworkV2Angular/src/app/post/post-comments/comment.ts
export interface Comment {
    id : number;
    date: Date;
    description: string;
    idPost: Number;
    user : string;
    UserImage: string;
}

export interface CommentCreation{
    description: string;
    date: Date;
    IdUser: number;
    IdPost: number;
}

File: MySocialNetworkV2Core/Entities/Comment.cs
using MySocialNetworkV2Core.Entities.CustomEntities;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MySocialNetworkV2Core.Entities
{
    public class Comment : BaseEntity
    {
        [Required]
        [MaxLength(500)]
        public string Description { get; set; }

        public DateTime Date { get; set; }
        [ForeignKey("IdPost")]
        public virtual Post Post { get; set; }
        public int IdPost { get; set; }
        [ForeignKey("IdUser")]
        public virtual User User { get; set; }
        public int IdUser { get; set; }


    }
}
